frontend/admin/src/js/components/user/UserProfileDocument.tsx

```diff
@@ -104,115 +100,122 @@ export const UserProfileDocument = React.forwardRef<
+                        })}
+                      </Heading>
+                    </HeadingWrapper>
+                    <WorkLocationSection applicant={applicant} />
+                  </div>
+                  <div className="page-section">
+                    <HeadingWrapper>
+                      <Heading icon={ThumbUpIcon} style={{ flex: "1 1 0%" }}>
+                        {intl.formatMessage({
+                          defaultMessage: "Work Preferences",
+                          description:
+                            "Title of the Work Preferences content section",
+                        })}
+                      </Heading>
+                    </HeadingWrapper>
+                    <WorkPreferencesSection applicant={applicant} />
+                  </div>
+                  <div className="page-section">
+                    <HeadingWrapper>
+                      <Heading
+                        icon={InformationCircleIcon}
+                        style={{ flex: "1 1 0%" }}
+                      >
+                        {intl.formatMessage({
+                          defaultMessage: "Employment Equity Information",
+                          description:
+                            "Title of the Employment Equity Information content section",
+                        })}
+                      </Heading>
+                    </HeadingWrapper>
+                    <DiversityEquityInclusionSection applicant={applicant} />
+                  </div>
+                  <div className="page-section">
+                    <HeadingWrapper>
+                      <Heading
+                        icon={CurrencyDollarIcon}
+                        style={{ flex: "1 1 0%" }}
+                      >
+                        {intl.formatMessage({
+                          defaultMessage: "Role and salary expectations",
+                          description:
+                            "Title of the Role and salary expectations section",
+                        })}
+                      </Heading>
+                    </HeadingWrapper>
+                    <RoleSalarySection applicant={applicant} />
+                  </div>
+                  <HeadingWrapper>
+                    <Heading
+                      icon={LightningBoltIcon}
+                      style={{ flex: "1 1 0%" }}
+                    >
+                      {intl.formatMessage({
+                        defaultMessage: "My skills and experience",
+                        description:
+                          "Title of the My skills and experience content section",
+                      })}
+                    </Heading>
+                  </HeadingWrapper>
+                  <ExperienceByTypeListing
+                    experiences={applicant.experiences?.filter(notEmpty)}
+                    defaultOpen
+                  />
+                </div>
+              );
+            })}
+          ;
```

Comment:
I think this is might be an accident
and is probably causing this orphaned semicolon here

![image](https://user-images.githubusercontent.com/40485260/179314399-8319fba9-6b56-4889-bb89-5ea17042c803.png)